utilities/mailer/index.mjs

```diff
@@ -36,12 +36,37 @@ export const sendMail = async (email, subject, message) => {
   htmlTemplate = htmlTemplate.replace("{{subject}}", subject)
   htmlTemplate = htmlTemplate.replace("{{messageBody}}", message)
 
+  // Stop execution if any of the required environment variables are not set
```

Comment:
I like this, good idea.

Just noting it does not impact /beta/bug or /beta/feedback, but it probably isn't meant to.